src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,36 @@
+package qna.domain;
+
+import qna.CannotDeleteException;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question", cascade = CascadeType.ALL)
+    private List<Answer> answers = new ArrayList<>();
+
+    protected Answers() {
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public List<DeleteHistory> delete(User loginUser) {
```
unchecked 검사로 만들어주세요